### PR TITLE
#4615 Fix text hover color change

### LINKED ISSUE #4615: [Community] Five and Friends: text color on hover of card - too low contrast
### Current Behavior
On this [page](https://layer5.io/community/adventures-of-five-and-friends), the text contrast is too low on hover in light mode.

### Desired Behavior
Make text white on hover.

### Screenshots / Mockups

https://github.com/layer5io/layer5/assets/7570704/83e7aa3d-5e73-4da0-9fd0-7f0670ef62d3

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by DkRadhey**
**Description**

This PR fixes #4615 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 9dcc69a2e95c2787a6016fd9887f2f0f078aceb0 at: https://64c0cfcef8f85037696985d2--layer5.netlify.app

**Comment by DkRadhey** [comment]
@l5io thanks

File: src/components/AdventuresVol/adventures-vol.style.js [hunk 1]
```diff
@@ -1,104 +1,110 @@
...
-	}
+    .handbook__card:hover {
+        scale: 1.07;
+    }
 
-	.handbook__card:hover .handbook__card--head {
-	  color: ${props => props.theme.text};
-	}
+    .handbook__card:hover .handbook__card--head {
+        color: white;
```
Comment (Chadha93) [inline src/components/AdventuresVol/adventures-vol.style.js:108]: @DkRadhey Use theme ref for white color.

**Comment by Ghat0tkach** [comment]
@DkRadhey 

Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Comment by DkRadhey** [comment]
The above link is asking for the permission. Please grant.

**Pull request closed without merging**